### DISCUSSION TITLE
Nick's ubeaut forestr

### DISCUSSION THREAD

**Discussion by kurtlawrence** [precision-mining-consulting/merz#17] [Ideas]
Let's make it gooey

**Comment by npdelaney** [comment]
![image](https://github.com/precision-mining-consulting/merz/assets/31809766/ce729434-8658-4c4c-a91b-190d35c6f658)

She slow, but at least it's honest work

**Comment by kurtlawrence** [comment]
'seeding'. nice.

I'm sure we can make it faster

**Comment by kurtlawrence** [comment]
From discussion @npdelaney:

1. Work on seeding techniques
2. Various tree models
3. Forest textures (dynamic tiling and colour grading?)
